packages/swing-store/src/transcriptStore.js

```diff
@@ -131,6 +140,14 @@ export function makeTranscriptStore(
     return transcripts;
   }
 
+  function* readVatTranscript(vatID) {
```

Comment:
We might call this `readFullVatTranscript` to make it clear that it's fetching multiple spans, which is not something we'd normally do for operational purposes. `readVatTranscript` isn't wrong.. it fits with our hierarchy of names:

* "transcript": everything from vat creation to vat termination (or latest delivery)
* "incarnation": everything from one upgrade to the next upgrade (or latest delivery, or termination)
* "span": everything from heap snapshot save/load to next upgrade, snapshot write, termination, or latest delivery
* "item" or "entry": one delivery

but "transcript" to mean "everything" is so rarely used, that it doesn't hurt to give it a longer name.